app/src/pages/inside/common/selectedItems/selectedItem/selectedItem.scss

```diff
@@ -1,5 +1,4 @@
 .selected-item {
-  height: 20px;
```

Comment:
Maybe it's better to leave this as `min-height`?

Reply:
have put it back as min-height